src/dmd/dtoh.d

```diff
@@ -188,53 +217,61 @@ private void initialize()
     }
 }
 
-extern(C++) void genCppHdrFiles(ref Modules ms)
+extern(C++) void genCppHdrFiles(ref Modules ms, const(char) *dir, const(char) *fname, bool only_c)
 {
     initialize();
 
     OutBuffer buf;
     buf.printf("// Automatically generated by %s Compiler v%d\n", global.vendor.ptr, global.versionNumber());
     buf.writeByte('\n');
-    buf.writestring("#pragma once\n");
```

Comment:
Why replace this? (Also this is causing the tests to fail.)

Reply:
#pragma once is a non-standard compiler extension, and the only reason it's used is to make humans' lives easier.  Computer-generated output can deal.

Reply:
Its supported by practically all major compilers. And you still need to change this, or change the tests to match.

Reply:
> Its supported by practically all major compilers.
And all the minor compilers (dmc) we care about for ci purposes.

Being backwards compatible with gcc-2.95 and the likes of others from that era is not a goal here.

Reply:
Given a choice between using a standard language feature and a nonstandard extension (which, incidentally, has been rejected from the standard for legitimate reasons), I cannot see a reason to choose the nonstandard extension over the standard language feature.

Reply:
`#pragma once` is a compile performance thing, that's why it's preferred. Is there any realistic potential targets where it's not supported?

Reply:
[No](https://en.wikipedia.org/wiki/Pragma_once#Portability)

Reply:
Compilers can detect header guards and optimize accordingly, so it doesn't affect performance.

Reply:
I you can name a compiler that doesn't support it, then you can change it, otherwise I'm going to block this PR in this issue.

Reply:
[Lacc](https://github.com/larmel/lacc).  [scc](http://www.simple-cc.org/), and [9cc](https://github.com/rui314/9cc) do not support it.

Reply:
9cc is no longer an active project, it successor still appears to be a personal project.

Lacc has a grand total of two contributors, with 99.7% of positive diff count going to one person.

acc has a few more but more that 76% of the commits are from one person.

Reply:
Your point being?  You asked me to cite c compilers, and I did.  If you'd like an example with more contributors, look at the plan9 c compiler.

Reply:
Maybe you'd be better to go and drop in a PR to support `#pragma once` into those projects :P

Reply:
I only consider tinycc to be a notable "other" C compiler... and that supports `#pragma once`. :-)